spec/c05recursion/generate/print_50_natural_nr_recursionSpec.js

```diff
@@ -0,0 +1,14 @@
+const printNaturalNumbers = require('../../../src/c05recursion/generate/print_50_natural_nr_recursion')
+
+describe('20 natural printNaturalNumbers', function () {
```

Comment:
name for describe is not suggestive describe('20 natural printNaturalNumbers', function () {